carbon-design-system/carbon
# docs(carbon-react): add storybook v6

Includes storybook v6 for the `@carbon/react` package. In order to get this working with hoisting in the monorepo, we had to inline the `storybook-preset-carbon` package in the `packages/react` folder 😞 

#### Changelog

**New**

- Add storybook v6 to `packages/carbon-react`

**Changed**

- Inline `storybook-preset-carbon` in `packages/react`

**Removed**

- `storybook-preset-carbon`

#### Testing / Reviewing

- Verify that the netlify previews succeed and deploy successfully
- Make sure `yarn storybook` works for `packages/react`
- Make sure `yarn storybook` works for `packages/carbon-react`
